go.mod

```diff
@@ -9,3 +9,5 @@ require (
 )
 
 replace github.com/kylelemons/godebug => github.com/jmillikin-stripe/godebug v0.0.0-20180620173319-8279e1966bc1
+
+go 1.13
```

Comment:
My editor or local build process seems to have added this automatically but I'll be honest I don't know why or if it's necessary. Please let me know if I should remove it from this PR.